authz/src/main/java/io/grpc/authz/FileWatcherAuthorizationServerInterceptor.java

```diff
@@ -47,20 +43,11 @@ public final class FileWatcherAuthorizationServerInterceptor implements ServerIn
       Logger.getLogger(FileWatcherAuthorizationServerInterceptor.class.getName());
 
   private volatile AuthorizationServerInterceptor internalAuthzServerInterceptor;
-  private final ScheduledExecutorService scheduledExecutorService = 
-      Executors.newSingleThreadScheduledExecutor(
-                    new ThreadFactoryBuilder()
-                    .setNameFormat("filewatcher" + "-%d")
-                    .setDaemon(true)
-                    .build());
 
   private final File policyFile;
-  private FileTime lastModifiedTime;
```

Comment:
FYI, for testing you can do something like:
https://github.com/grpc/grpc-java/blob/1551cc7403e0f9bcb3b949be8c486ce207395c6b/xds/src/test/java/io/grpc/xds/internal/security/certprovider/FileWatcherCertificateProviderTest.java#L117-L118